mungegithub/mungers/publish_scripts/apimachinery_sync_from_kubernetes.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
I don't follow the argument. I think what would impede contribution is that the cluster is running in a google provided cluster, not the location of the scripts.